EventPlaning.BL.Tests/EventManagerTests.cs

```diff
@@ -99,7 +111,7 @@ public void RemoveActivityOK()
 
             eventManager.RemoveActivityFromEvent(1, activity);
 
-            Assert.AreEqual(1, eventManager.GetEventActivities(1).Count);
```

Comment:
Would say that is beneficial to check the count or the value is not null, because it is important